lib/_stream_readable.js

```diff
@@ -764,12 +764,12 @@ function flow(stream) {
 // wrap an old-style stream as the async data source.
 // This is *not* part of the readable stream interface.
 // It is an ugly unfortunate mess of history.
-Readable.prototype.wrap = function(stream) {
+Readable.prototype.wrap = function wrap(stream) {
   var state = this._readableState;
   var paused = false;
 
   var self = this;
-  stream.on('end', function() {
+  stream.on('end', function end() {
```

Comment:
I wasn't sure which name could be more useful, so this comments are great, thanks